lib/install/validate/validator_test.go

```diff
@@ -250,25 +250,77 @@ func testStorage(v *Validator, input *data.Data, conf *config.VirtualContainerHo
-		{"LocalDS_0/images", map[string]string{"volume1": "LocalDS_0/volumes/volume1", "volume2": "ds://LocalDS_0/volumes/volume2"}, false, "ds://LocalDS_0", map[string]string{"volume1": "ds://LocalDS_0/volumes/volume1", "volume2": "ds://LocalDS_0/volumes/volume2"}},
-		{"ds://LocalDS_0/images/xyz", map[string]string{"volume1": "LocalDS_0/volumes/volume1", "volume2": "ds://LocalDS_0/volumes/volume2"}, false, "ds://LocalDS_0", map[string]string{"volume1": "ds://LocalDS_0/volumes/volume1", "volume2": "ds://LocalDS_0/volumes/volume2"}},
-		{"LocalDS_0", map[string]string{"volume1": "LocalDS_1/volumes/volume1", "volume2": "ds://LocalDS_1/volumes/volume2"}, true, "", nil},
-		{"ds://LocalDS_0", map[string]string{"volume1": "LocalDS_1/volumes/volume1", "volume2": "ds://LocalDS_1/volumes/volume2"}, true, "", nil},
-		{"", map[string]string{"volume1": "", "volume2": "ds://"}, true, "", nil},
-		{"ds://", map[string]string{"volume1": "", "volume2": "ds://"}, true, "", nil},
+		{"LocalDS_0",
+			map[string]string{"volume1": "LocalDS_0/volumes/volume1",
+				"volume2": "ds://LocalDS_0/volumes/volume2"},
+			false,
+			"ds://LocalDS_0/test001/images",
+			map[string]string{"volume1": "ds://LocalDS_0/volumes/volume1",
+				"volume2": "ds://LocalDS_0/volumes/volume2"}},
+
+		{"LocalDS_0/images",
+			map[string]string{"volume1": "LocalDS_0/volumes/volume1",
+				"volume2": "ds://LocalDS_0/volumes/volume2"},
+			false,
+			"ds://LocalDS_0/images",
+			map[string]string{"volume1": "ds://LocalDS_0/volumes/volume1",
+				"volume2": "ds://LocalDS_0/volumes/volume2"}},
+
+		{"ds://LocalDS_0/images",
+			map[string]string{"volume1": "LocalDS_0/volumes/volume1",
+				"volume2": "ds://LocalDS_0/volumes/volume2"},
+			false,
+			"ds://LocalDS_0/images",
+			map[string]string{"volume1": "ds://LocalDS_0/volumes/volume1",
+				"volume2": "ds://LocalDS_0/volumes/volume2"}},
+
+		{"ds://LocalDS_0/images/xyz",
+			map[string]string{"volume1": "LocalDS_0/volumes/volume1",
+				"volume2": "ds://LocalDS_0/volumes/volume2"},
+			false,
+			"ds://LocalDS_0/images/xyz",
+			map[string]string{"volume1": "ds://LocalDS_0/volumes/volume1",
+				"volume2": "ds://LocalDS_0/volumes/volume2"}},
+
+		{"ds://LocalDS_0",
+			map[string]string{"volume1": "LocalDS_1/volumes/volume1",
+				"volume2": "ds://LocalDS_1/volumes/volume2"},
+			true,
+			"ds://LocalDS_0/test001/images",
+			nil},
+
+		{"LocalDS_0",
+			map[string]string{"volume1": "LocalDS_1/volumes/volume1",
+				"volume2": "ds://LocalDS_1/volumes/volume2"},
+			true,
+			"ds://LocalDS_0/test001/images",
+			nil},
+
+		{"",
+			map[string]string{"volume1": "",
+				"volume2": "ds://"},
+			true,
+			"",
+			nil},
+
+		{"ds://",
+			map[string]string{"volume1": "",
+				"volume2": "ds://"},
+			true,
+			"",
+			nil},
```

Comment:
can we have a datastore with a space in the name - given I failed to anticipate that initially having a test to confirm we don't break it seems a good idea.